docs/getting_started/setup_your_environment.md

```diff
@@ -98,6 +98,11 @@ Please report any issues.
 Support for JetBrains IDEs is available through
 [the Deno plugin](https://plugins.jetbrains.com/plugin/14382-deno).
 
+Once installed, replace the content of 
+`External Libraries > Deno Library > lib > lib.deno.d.ts` with the output of 
+`deno types`. that will fix outdates typings, making it usable for current 
+versions. You will have to do this every time you change deno version.
+
```

Comment:
```suggestion
Once installed, replace the content of 
`External Libraries > Deno Library > lib > lib.deno.d.ts` with the output of 
`deno types`. This will ensure the typings for the extension match the current
version.  You will have to do this every time you update the version of Deno.
```